app/assets/javascripts/patients.coffee

```diff
@@ -91,3 +91,12 @@ $(document).on 'turbolinks:load', ->
       placement: 'top',
       title: $(@).data( 'tooltip-text' )
     } )
+
+
+  $( '.tooltip-header' ).parent( '.checkbox' ).each ->
```

Comment:
Here's what I came up with after hacking on this for a bit -- this seems to work but it's pretty ugmo -- 

```
  $( '.tooltip-header' ).parent().find('.has-tooltip, .form-control').each ->
    $(@).parent().find( 'span.tooltip-header-help' ).tooltip( {
      html: true,
      placement: 'bottom',
      title: $(@).data( 'tooltip-text' )
    } )
    $(@).parents('.form-group').find('.tooltip-header-help').tooltip( {
      html: true,
      placement: 'bottom',
      title: $(@).data( 'tooltip-text' )
    } )
```

It looks like the DOM structure of `bootstrap_form_for`'s checkboxes and text inputs are different, so I split the difference, selected them all thru, and then just call `tooltip` on empty elements. I'm sure that's a javascript venial sin but I couldn't think of a good way to do `parents with class OR parent` and have it blend cleanly.

You'll also note that this depends on the checkboxes having a `has-tooltip` class. A more optimal way to do that would be to collapse the first line there to something like this:

```
  $( '.tooltip-header' ).parent().find('input').filter((x) -> x.attr('data-tooltip-text') != "").each -> ....
```

but I couldn't figure out the magic invocation.

tldr: This will work as is if you paste it in and slap 'has-tooltip' on the input elements in the views!